qcow2reader_test.go

```diff
@@ -1,26 +1,21 @@
-package qcow2reader
+package qcow2reader_test
```

Comment:
The test should use the same package as the code, no?

Reply:
This is for whitebox tests that need access to the internals. For blackbox tests, it is better to have the test in the special *_test package. This way we use the code in the same way a user would use the code.

I think the default should be blackbox test, and whitebox tests should be used only if needed.

https://github.com/golang/go/issues/25223

Reply:
Thanks, could you add that context as a comment?

Reply:
Sure, will add.

Reply:
Add a package comment.